get_pdfs.go

```diff
@@ -82,18 +97,42 @@ func (u FileDownloader) Perform() {
 		if err == nil {
 			err = cerr
 		}
-
 		if size < 49000 || size > 80000 {
 			e := os.Remove(u.FileName)
 			if e != nil {
 				log.Fatal(e)
 			}
+		} else {
+			convertCSV(u.FileName, u.FileNameCSV)
 		}
-
-		//do csv conversion
-		
-		//close
 	}()
 
 	fmt.Printf("Downloaded a file %s with size %d\n", u.FileName, size)
 }
+
+func convertCSV(filename string, FileNameCSV string) {
```

Comment:
this should be in the Perform method of the FileConverter

Reply:
I faced problems with doing that because of deferring the close of the file. So that means I have to use another closure in there to handle deferring...

Reply:
Moved it to a closure inside the perform